documentation/JavaFiddle_Requirements.txt

```diff
@@ -58,5 +58,8 @@ Reviewboard functionality
 - add/edit/read comments
 - approve/reject
 
+[Tsanda Artyom] Ideas:
+- add possibility to import project from an existing one on the local machine
```

Comment:
It is quite complicated. We have to analyze structure of the project, also it can be used for system hacking. An issue to discuss